### PR TITLE
Add Brainfuck support

### PR DESCRIPTION
**Pull request by sjparkinson**
Includes bf ('Yet  another  Brainfuck interpreter'), a simple interpreter. Built by Stephan Beyer, the source is avaliable from http://pkqs.net/~sbeyer/.

Use `man bf` to get started.

boom

---

Thanks to @burtonageo for the suggestion at yesterdays meetup :smile:.

### CODE REVIEW

File: brainfuck/Vagrantfile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure("2") do |config|
+
+    hostname = "csharp.box"
```
Comment (adlawson) [inline brainfuck/Vagrantfile]: Could you update this field to `brainfuck.box`

**Comment by adlawson** [comment]
Could you also add links etc to the table in the Readme.
Once that's all in I'll have another look, but it should be good to merge then :+1: 

Thanks, Sam!

**Comment by adlawson** [comment]
I also read the title and was all

**Comment by sjparkinson** [comment]
> I also read the title and was all

ironic

**Comment by adlawson** [comment]
Good work. Keep em coming :rocket:

**Pull request merged**